packages/polar/src/lib/response.ts

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
please add doc string here about the function

Reply:
done